core/src/main/java/org/lflang/generator/c/CTriggerObjectsGenerator.java

```diff
@@ -157,14 +118,29 @@ public static String generateSchedulerInitializer(
             "                        .num_reactions_per_level = &num_reactions_per_level[0],",
             "                        .num_reactions_per_level_size = (size_t) "
                 + numReactionsPerLevel.length
-                + "};",
-            "lf_sched_init(",
-            "    (size_t)_lf_number_of_workers,",
-            "    &sched_params",
-            ");"));
+                + "};"));
+
+    for (ReactorInstance enclave : CUtil.getEnclaves(main)) {
+      code.pr(generateSchedulerInitializerEnclave(enclave, targetConfig));
+    }
+
     return code.toString();
   }
 
+  // FIXME: Probably we want some enclaveConfig handed off
```

Comment:
I do not totally understand the FIXMEs around this method. In any case we can just remember to open issues for them when this is merged.